tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/Properties/InterceptingProjectProperties/BuildPropertyPage/NameQuotedValuePairListEncodingTests.cs

```diff
@@ -36,6 +36,9 @@ public class NameQuotedValuePairListEncodingTests
         [InlineData("\"\"ab\"\"cd=     value1     ", "/\"/\"ab/\"/\"cd=\"     value1     \"")]
         [InlineData("\"\"ab\"\"cd=value1     ", "/\"/\"ab/\"/\"cd=\"value1     \"")]
         [InlineData("\"\"ab\"\"cd=     value1", "/\"/\"ab/\"/\"cd=\"     value1\"")]
+        [InlineData("", "")]
+        [InlineData(",", "")]
+        [InlineData(",,,", "")]
```

Comment:
The first of these three cases makes sense to me, but the second two are less clear. They are still potentially invalid strings that should raise format exceptions.

I think the implementation changes could be reduced in this PR to just checking for an empty string in `Parse` and returning and empty enumeration in such cases.